packages/firefuel/lib/src/firefuel_collection.dart

```diff
@@ -8,7 +8,8 @@ abstract class FirefuelCollection<T extends Serializable>
 
   final firestore = Firefuel.firestore;
 
-  FirefuelCollection(this.path);
+  FirefuelCollection(String path, {bool useEnv = true})
+      : this.path = '${useEnv ? Firefuel.env : ''}$path';
```

Comment:
This is a good one liner, but I think we're using a ternary when we shouldn't be. Semantically a ternary is supposed to mean that it should always do one or the other, where I think this should only ever be doing the one thing, or nothing.

How do you feel about pulling this functionality into a static function to build the environmental path?

With a static function you can just do an if statement to add the env, or return the original path.

Open to feedback here.

Reply:
It will make it easier to read to have a static function. Let's go with that